lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -158,11 +186,13 @@ def self.get_or_create_app(params)
       def self.run(params)
         values = params.values
         upload_dsym_only = params[:upload_dsym_only]
+        upload_mapping_only = params[:upload_mapping_only]
 
         # if app found or successfully created
         if self.get_or_create_app(params)
-          self.run_release_upload(params) unless upload_dsym_only
-          self.run_dsym_upload(params)
+          self.run_release_upload(params) unless upload_dsym_only unless upload_mapping_only
+          self.run_dsym_upload(params) 
```

Comment:
Shouldn't this be skipped if `upload_mapping_only`?

Reply:
yep, make sense